front/src/modules/timetableItem/components/ImportTimetableItem/helpers/parseXML.ts

```diff
@@ -138,11 +347,11 @@ const parseXML = async (xmlDoc: Document): Promise<TimetableJsonPayload> => {
       train_name: trainNumber,
       rolling_stock_name: rollingStockName || formationRef || '', // RollingStocks in xml files rarely have the correct format
       start_time: new Date(`${startDate} ${firstDepartureTimeformatted}`).toISOString(),
-      constraint_distribution: xmlDoc.documentElement.hasAttribute('xmlns:tps')
-        ? 'STANDARD'
-        : 'MARECO',
```

Comment:
Why remove this?

Reply:
I think @achrafmohye made a typo in his comment: the code switches this to `STANDARD` unconditionally.

Cool, if that's something discussed with POs, that's completely fine by me - even allows us to remove a special case! (Ideally should be on its own commit.)